packages/graphql/src/schemaTypes/objectTypes/gql-CurrentProject.ts

```diff
@@ -135,7 +135,8 @@ export const CurrentProject = objectType({
       },
     })
 
-    t.nonNull.string('defaultSpecFileName', {
+    // nullable so we can throw if it can't be decided
+    t.string('defaultSpecFileName', {
       description: 'Default spec file name for spec creation',
```

Comment:
```suggestion
    t.string('defaultSpecFileName', {
      description: 'Default spec file name for spec creation, nullable / throws if it can't be decided',
```

Reply:
hahahaha

Reply:
unterminated string literal